[2024-11-03T04:04:44.196+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-03T04:04:44.203+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.create_table_user_review manual__2024-11-03T04:04:41.977363+00:00 [queued]>
[2024-11-03T04:04:44.207+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.create_table_user_review manual__2024-11-03T04:04:41.977363+00:00 [queued]>
[2024-11-03T04:04:44.207+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-11-03T04:04:44.212+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): create_table_user_review> on 2024-11-03 04:04:41.977363+00:00
[2024-11-03T04:04:44.216+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=74) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-03T04:04:44.217+0000] {standard_task_runner.py:72} INFO - Started process 82 to run task
[2024-11-03T04:04:44.218+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'create_table_user_review', 'manual__2024-11-03T04:04:41.977363+00:00', '--job-id', '249', '--raw', '--subdir', 'DAGS_FOLDER/***.py', '--cfg-path', '/tmp/tmpokanz0oy']
[2024-11-03T04:04:44.220+0000] {standard_task_runner.py:105} INFO - Job 249: Subtask create_table_user_review
[2024-11-03T04:04:44.259+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline.create_table_user_review manual__2024-11-03T04:04:41.977363+00:00 [running]> on host f35298df57b7
[2024-11-03T04:04:44.304+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='your_name' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='create_table_user_review' AIRFLOW_CTX_EXECUTION_DATE='2024-11-03T04:04:41.977363+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-03T04:04:41.977363+00:00'
[2024-11-03T04:04:44.305+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-03T04:04:45.624+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-03T04:04:45.667+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-03T04:04:45.669+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=create_table_user_review, run_id=manual__2024-11-03T04:04:41.977363+00:00, execution_date=20241103T040441, start_date=20241103T040444, end_date=20241103T040445
[2024-11-03T04:04:45.728+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-03T04:04:45.750+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
